### PR TITLE
Update nace_classification for automotive sales to have borderline = TRUE

### LINKED ISSUE #213: Some `nace_classification` borderline sectors missing `borderline = TRUE` flag
45.1, 45.11 and 45.19 are all related to the sale of motor vehicles. I believe that, however, owing to the misclassification of many loans, these should still be included as potentially having production, and should be identified with the `borderline = TRUE` flag. 

Currently, the flag is `FALSE`

``` r
library(dplyr)
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union
library(r2dii.data)

r2dii.data::nace_classification %>% 
  filter(sector == "automotive")
#> # A tibble: 6 x 5
#>   original_code code  code_level sector     borderline
#>   <chr>         <chr>      <dbl> <chr>      <lgl>     
#> 1 29.10         2910           4 automotive FALSE     
#> 2 29.20         2920           4 automotive TRUE      
#> 3 29.32         2932           4 automotive TRUE      
#> 4 45.1          451            3 automotive FALSE     
#> 5 45.11         4511           4 automotive FALSE     
#> 6 45.19         4519           4 automotive FALSE
```

<sup>Created on 2021-02-11 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

### PR DESCRIPTION
**Pull request by jdhoffa**
Closes #213

### CODE REVIEW

File: tests/testthat/test-sector_classifications.R [hunk 1]
```diff
@@ -6,3 +6,15 @@ test_that("has an entry in data_dictionary", {
...
+  automotive <- c(451, 4511, 4519)
+  borderline <- subset(
+    sector_classifications,
+    subset = code_system == "NACE" & code %in% automotive,
+    select = "borderline",
+    drop = TRUE
+  )
+
+  expect_true(all(borderline))
+})
```
Comment (maurolepore) [inline tests/testthat/test-sector_classifications.R:20]: I rewrote this a bit.

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # r2dii.data (development version)
 
+* In `nace_classification`, where codes belong to automotive sales all values of
+`borderline` are now `TRUE` (Jakub Lewandowski #213).
```
Comment (maurolepore) [inline NEWS.md:4]: I added an entry to NEWS.

**Review by maurolepore — APPROVED** [review]
Thanks Jackson!
I did two minor contributions :-)

File: NEWS.md [hunk 3]
```diff
@@ -1,9 +1,12 @@
 # r2dii.data (development version)
 
+* NACE codes for automotive sales now have `borderline` correctly set to `TRUE` 
+  (@KapitanKombajn #213).
```
Comment (jdhoffa) [inline NEWS.md]: Added Jakub's github account

File: NEWS.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # r2dii.data (development version)
 
-* NACE codes for automotive sales now have `borderline` correctly set to `TRUE` 
-  (@KapitanKombajn #213).
+* `nace_classification` codes for automotive sales now have `borderline` values 
+  of `TRUE` (@KapitanKombajn #213).
```
Comment (jdhoffa) [inline NEWS.md:4]: Tagged Jakub's github account.

**Pull request merged**